packages/e2e/features/ui/components/authenticator/sign-in-with-username.feature

```diff
@@ -17,6 +17,20 @@ Feature: Sign In with Username
     And I click the "Sign in" button
     Then I see "User does not exist"
 
+  @angular @react @vue
+  Scenario: Verify Submit text is correct on confirm Reset Password Page without translation
+    And I click the "Forgot your Password?" button
+    When I type my "username" with status "CONFIRMED"
+    And I intercept '{ "headers": { "X-Amz-Target": "AWSCognitoIdentityProviderService.ForgotPassword" } }' with fixture "reset-password"
+    And I click the "Send code" button
+    Then I will be redirected to the confirm forgot password page
+    And I see "Code"
+    Then I type a valid code
+    And I type my new password
+    And I confirm my password
+    And I intercept '{ "headers": { "X-Amz-Target": "AWSCognitoIdentityProviderService.ConfirmForgotPassword" } }' with fixture "confirm-reset-password"
+    And I see "Submit"
```

Comment:
This test verifies that we indeed see `Submit` for the base case, instead of `Send Code`. This test has no translations of `Send Code`.